shared/cfg/guest-os/Linux/Fedora/27.cfg

```diff
@@ -0,0 +1,83 @@
+            unattended_install.cdrom:
+                md5sum_cd1 = 1a9cb7c3feeb5a6d5c4319dd9b60aa93
+                md5sum_1m_cd1 = 061c5f6acec56e5b7defd632de673349
+            unattended_install.url:
+                sha1sum_vmlinuz = 76736a88e41fc83b7f0962cc3a72fbddd1c61a3f
+                sha1sum_initrd = 39f485b4ee63d03fe9ef7c9448c8e453616a352c
+        ppc64:
+            kernel_params = "console=hvc0 serial"
+            boot_path = ppc/ppc64
+            unattended_install.cdrom:
+                md5sum_cd1 = 0572be417a7c7643b60a46599fc631ae
+                md5sum_1m_cd1 = e76f90cc0e3b292e14a066cf3e07e2dc
+            unattended_install.url:
+                sha1sum_vmlinuz = a95fa545e70d71d6cdfc77aab8d01c76
+                sha1sum_initrd = b6c8313fb6e469235c1489617c8339fa
+        ppc64le:
+            kernel_params = "console=hvc0 serial"
+            boot_path = ppc/ppc64
+            unattended_install.cdrom:
+                md5sum_cd1 = cc64505fe4721a38bb991fbb1c65064b
+                md5sum_1m_cd1 = 23692021947b6cb0e2c0be85be931b88
+            unattended_install.url:
+                sha1sum_vmlinuz = 53830fc2db8b09f1eda4e500f6291e9c
+                sha1sum_initrd = 3a406f1a2a39488a5073ae19db294413
+        s390x:
+            kernel_params = "console=ttysclp0 serial"
+            boot_path = images
+            kernel = images/f27-s390x/kernel.img
+            unattended_install.cdrom:
+                md5sum_cd1 = ca754437f6800bcb5dd252fb4c4a4251
+                md5sum_1m_cd1 = 38832089d02673721177c3e34375e73e
+            unattended_install.url:
+                sha1sum_vmlinuz = f94cadb2293aaf255bdbba0fd674762c
+                sha1sum_initrd = fe4fa07235a5dcf8f942ba57c165b031
+        x86_64:
+            kernel_params = "console=tty0 console=ttyS0"
+            unattended_install.url:
+                url = http://dl.fedoraproject.org/pub/fedora/linux/releases/27/Server/${vm_arch_name}/os
+            unattended_install.cdrom:
+                md5sum_cd1 = f148145afbccebe532dc710362352052
+                md5sum_1m_cd1 = 1d57d344674dc84c08eca4ebcd0269b5
+            unattended_install.url:
+                sha1sum_vmlinuz = b72b10be9d895d8f867e1301c598d4c1
+                sha1sum_initrd = 6b465695c57b9a8dac4ec3afc86e7b23
+        # Shared specific setting
+        unattended_install.url:
+            # Installation works fine with mem=1024 on methods such as cdrom
+            # but fails ("No space left on device") with methods such as url.
+            mem = 2048
+            # Unless overridden use secondary url, because it's the most common one
+            url ?= http://dl.fedoraproject.org/pub/fedora-secondary/releases/27/Server/${vm_arch_name}/os/
+            kernel_params += " inst.repo=${url}"
+        unattended_install.cdrom:
+            cdrom_cd1 = isos/linux/Fedora-Server-dvd-${vm_arch_name}-27-1.6.iso
+        extra_cdrom_ks:
+            kernel_params += " ks=cdrom"
```

Comment:
is this line should be ks=cdrom:/ks.cfg'?

Reply:
This is copied from F25 and also from JeOS. I remember on some versions some variants were unacceptable and this always worked well.